vespajlib/src/main/java/com/yahoo/io/NativeIO.java

```diff
@@ -0,0 +1,87 @@
+package com.yahoo.io;
+
+import java.io.File;
+import java.io.FileDescriptor;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.logging.Logger;
+
+import com.sun.jna.LastErrorException;
+import com.sun.jna.Native;
+import com.sun.jna.Platform;
+
+public class NativeIO {
+    private final Logger logger = Logger.getLogger(getClass().getName());
```

Comment:
Why is the logger not static?

Reply:
Probably because it was not static where I copied the initial code from, which in this case was Searcher.java
Fixed.